signal_your_workflow/external_signal_main.py

```diff
@@ -0,0 +1,26 @@
+import asyncio
+
+from temporalio.client import Client
+from signal_external_wf_dacx import WorkflowA, WorkflowB
+
+
+async def main():
+    client = await Client.connect("localhost:7233")
+    handle_b = await client.start_workflow(
```

Comment:
Is this workflow gonna remain open on a user's server? May want to complete/cancel/terminate it